packages/db/src/types.ts

```diff
@@ -1,10 +1,10 @@
 export interface AsyncOperationRecord {
```

Comment:
I think that perhaps these types should be included in the @cumulus/types packages (again referring to #1992 for discussion of the counter implementation)

Reply:
Based on review commentary in #1992 I'm going to resolve this, given that decision is already made - we can address this outside the scope of the PR should it be needed.